kore/src/Kore/Internal/SideCondition.hs

```diff
@@ -292,7 +324,7 @@ toPredicate
     :: InternalVariable variable
     => SideCondition variable
     -> Predicate variable
-toPredicate condition@(SideCondition _ _) =
+toPredicate condition@(SideCondition _ _ _) =
```

Comment:
I think the `replacements` always come from the `assumedTrue`, so they can be omitted here, but shouldn't the `definedTerms` be included?